docs/team/yijiano.md

```diff
@@ -0,0 +1,120 @@
+# Zhang YIjian - Project Portfolio Page
+
+## Overview
+Pharmacy Inventory & Logistics Ledger (PILL) is a desktop application that allows
+pharmacists to keep track of and manage medicinal inventory.
+
+PILL is fully written in Java, and users can interact with the application using
+Command Line Interface (CLI).
+
+### Summary of Contributions
+
+- **New Feature**: Added User Interface for the application (Ui Class)
+    - What it does: The Ui class is responsible for handling the user interface of the application ([#39](https://github.com/AY2425S1-CS2113-W14-4/tp/pull/39)).
+    - Justification: Abstraction of the user interface from the main logic of the application.
+    - Highlights: ASCII Art, welcome message, exit message, error messages and command prompt.
+- **New Feature**: Added parser for commands.
+    - What it does: Abstracts the parsing of user input into commands ([#39](https://github.com/AY2425S1-CS2113-W14-4/tp/pull/39)).
+    - Justification: Separation of concerns between user input and command execution.
+    - Highlights: Handles all exceptions and errors within the Parser class, allowing for cleaner code in the main logic.
+- **New Feature**: Added custom exceptions for the application ([#39](https://github.com/AY2425S1-CS2113-W14-4/tp/pull/39)).
+    - What it does: Allows us to handle exceptions in a more structured manner.
+    - Justification: Allows for more specific error messages to be displayed to the user.
+    - Highlights: Custom messages that better fit the theme and are more in-line with other flavour text can be implemented.
```

Comment:
Should this be shorter? I feel that more complex/difficult to implement features should take more space in your PPP, to reflect their relative contribution